server-config.json

```diff
@@ -5,6 +5,7 @@
 	"dbconfig": "./focalboard.db",
 	"postgres_dbconfig": "dbname=focalboard sslmode=disable",
 	"useSSL": false,
+    "secureCookie": false,
```

Comment:
This is probably ok for now but in the future we could automatically set the secure cookie attribute based on HTTP headers like we do in the MM server https://github.com/mattermost/mattermost-server/blob/9cc5089af8b7382c0170417bd2c1808662000573/app/login.go#L243